c10/DeviceType.cpp

```diff
@@ -1,9 +1,9 @@
-#include <ATen/core/DeviceType.h>
+#include <c10/DeviceType.h>
 #include <c10/util/Exception.h>
 
-namespace at {
+namespace c10 {
 
-std::string DeviceTypeName(at::DeviceType d, bool lower_case) {
+std::string DeviceTypeName(DeviceType d, bool lower_case) {
```

Comment:
nit: in line 27-30, we still mention caffe2.proto which we might want to just remove? :)